lib/util/priv.ex

```diff
@@ -1,6 +1,6 @@
 defmodule Util.Priv do
   def get_priv_path(filename) do
```

Comment:
maybe change the name of this module and fn? Something like `Util.Data.get_path` or similar

Reply:
resolved